lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -937,10 +941,14 @@ def test_filestream_tarfile_open(self, monkeypatch, tmp_path):
         def fake_tarfile_open(self, path):
             raise tarfile.TarError("Invalid Tarfile")
 
+        def mock_open(self, mode="rb"):
+            raise FileNotFoundError(f"No such file or directory: '{tar}'")
```

Comment:
How could it be that we have a dataset whose tarball cannot be found/opened?  It seems like we should either fail to find the dataset or the tarball should be there...it shouldn't be the case that we have one without the other...right??